src/AccessibilityInsights.SharedUx/Controls/TestTabs/ColorContrast.xaml.cs

```diff
@@ -119,13 +120,29 @@ private void RunAutoCCA(Bitmap bitmap)
 
             if (pair == null)
             {
+                Logger.PublishTelemetryEvent(
```

Comment:
Should we be using a `TelemetryEventFactory` instead? I couldn't find one to extend that appeared in-scope here.

Reply:
It would be good to have, but I'd argue that it belongs in a separate PR that leverages it broadly